### ISSUE TITLE
2.1 Accessibility Services of Platform Software

### ISSUE THREAD

**Issue by melaniephilipp** [w3c/wcag2ict#228]
Can examples or information about how to view/expose/interact with the accessibility API provided by software be provided? This also relates to Note 1 for 1.3.1.

**Comment by maryjom** [comment]
Thank you for taking the time to review the WCAG2ICT public draft. Our task force will review your comment and develop a response that we hope will address your concern. The response will be drafted in a comment on this issue, marked **DRAFT RESPONSE** until the Task Force reaches consensus.

**Comment by mitchellevan** [comment]
My perspective (not Working Group consensus):

- Examples of accessibility APIs can be found at [HTML Accessibility API Mappings (HTML-AAM)](https://www.w3.org/TR/html-aam/). User agents and non-web software that runs on these platforms can also use these same accessibility APIs.
- Other platforms with many form factors, from smart TVs to watches and everything in between, may have their own versions of accessibility APIs, including platforms that do not have a browser.

If we added something like the above points to [WCAG2ICT](https://www.w3.org/TR/wcag2ict-22/), would it address your question?

If WCAG2ICT even tried to list the possible accessibility APIs, it would be incomplete and it wouldn't age well. And it's out of scope for this TAsk Force to provide anything like the kinds of details in HTML-AAM.

**Comment by melaniephilipp** [comment]
Thank you @mitchellevan. Exposing the accessibility API for web content is simple using browser inspect tools. We are looking for examples of how one might view/expose/interact with the accessibility API provided by software (and other non-web ICT) for testing 1.3.1 and other relevant SCs.  Thank you

**Comment by mraccess77** [comment]
Tools such as Microsoft's Accessibility Insights or XCode's Accessibility Inspector can be used to inspect APIs on their respective open platforms.

**Comment by maryjom** [comment]
Regarding others' comments, some native software accessibility API inspection tools require you to be a developer working on the source code and using the development tools. A test professional that is testing the built and delivered product software might have to resort instead to completely testing manually and using a screen reader.

**Comment by maryjom** [comment]
DRAFT RESPONSE: 
This task force is narrowly scoped to providing interpretation of WCAG in a non-web context. For the content in the WCAG2ICT Note, we are limited to that scope and not at liberty to investigate and document available tooling to be used for accessibility testing in a non-web environment. Just as we cannot develop techniques for non-web software or documents to conform with WCAG. We do feel such resources would be very helpful for those working on and testing non-web documents and software for conformance to WCAG criteria. Development of such resources could be done by establishing a W3C Community Group.

**Comment by maryjom** [comment]
Closing as answered.
